user-service/src/main/resources/application-local.yml

```
@@ -28,4 +31,22 @@ management:
+        sliding-window-size: 10
+        record-exceptions:
+          - java.io.IOException
+          - feign.RetryableException
+          - feign.FeignException.FeignServerException
+        ignore-exceptions:
+          - feign.FeignException.FeignClientException
+  timelimiter:
+    configs:
+      default:
+        timeout-duration: 5s
+        cancel-running-future: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)